gulpfile.js

```diff
@@ -1,4 +1,4 @@
-﻿const gulp = require('gulp');
```

Comment:
weird... an ascii conversion says there are three hidden characters here that got removed.  Probably a good thing.